mbed_lstools/lstools_base.py

```diff
@@ -73,6 +73,8 @@ def __init__(self):
         self.lock_file = os.path.join(MbedLsToolsBase.HOME_DIR,
                                       MbedLsToolsBase.MBEDLS_HOME_DIR, MbedLsToolsBase.MBEDLS_GLOBAL_LOCK)
         self.mbedls_get_mocks()
+        # Make sure retargeting is applied if retarget file exists
+        self.retarget(True)
```

Comment:
@stevew817 Can you please revert your commit https://github.com/ARMmbed/mbed-ls/pull/146/commits/ccc88820ed9008685140ba82970a922c697921e5. Your previous changes are good to be merged.